src/templates/common/medallia/index.test.tsx

```diff
@@ -20,8 +15,78 @@ jest.mock('@/lib/utils/medallia', () => ({
   setWindowVaSurvey: jest.fn(),
 }))
 
+// Mock recordEvent
+jest.mock('@/lib/analytics/recordEvent', () => ({
+  recordEvent: jest.fn(),
+}))
+
+interface MedalliaEvent extends CustomEvent {
+  detail: {
+    Form_Type?: string
+    Form_ID?: string
+    Feedback_UUID?: string
+    Content?: string
+  }
+}
+
+const mockEvent: MedalliaEvent = {
+  detail: {},
+  initCustomEvent: function (
+    type: string,
+    bubbles?: boolean,
+    cancelable?: boolean,
+    detail?: unknown
+  ): void {
+    throw new Error('Function not implemented.')
+  },
+  bubbles: false,
+  cancelBubble: false,
+  cancelable: false,
+  composed: false,
+  currentTarget: undefined,
+  defaultPrevented: false,
+  eventPhase: 0,
+  isTrusted: false,
+  returnValue: false,
+  srcElement: undefined,
+  target: undefined,
+  timeStamp: 0,
+  type: '',
+  composedPath: function (): EventTarget[] {
+    throw new Error('Function not implemented.')
```

Comment:
😂 That works, I guess.

Reply:
my fiend vscode `add missing properties` added all these missing properties for me, so I can't take credit for that 😄